### DISCUSSION TITLE
How to determine what version of the theme do I have?

### DISCUSSION THREAD

**Discussion by arvera** [mmistakes/minimal-mistakes#2524] [Q&A] [answered]
I saw documentation that talks about the migration. I am interested on getting the search with google working for my blog site, and because the documentation here: https://mmistakes.github.io/minimal-mistakes/docs/layouts/

talks about: 

> Note: A page using the layout: search isn’t compatible with the new site search feature incorporated in the masthead.

I started to wonder what version of the them do I have.

**Comment by mmistakes** [comment, marked as answer]
https://mmistakes.github.io/minimal-mistakes/docs/upgrading/

View source of a page on your site. You’ll see something like this near the top that has the version number.

```html
<!--
  Minimal Mistakes Jekyll Theme 4.19.2 by Michael Rose
  Copyright 2013-2019 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE
-->
```
